loader.rb

```diff
@@ -9,15 +9,72 @@
 
 require "bundler/setup"
 Bundler.setup
-require "zeitwerk"
-Loader = Zeitwerk::Loader.new
-Loader.push_dir("#{__dir__}/")
-Loader.push_dir("#{__dir__}/model")
-Loader.push_dir("#{__dir__}/lib")
-Loader.ignore("#{__dir__}/routes")
-Loader.ignore("#{__dir__}/migrate")
-Loader.ignore("#{__dir__}/spec")
-Loader.ignore("#{__dir__}/model.rb")
-Loader.inflector.inflect("db" => "DB")
-Loader.enable_reloading
-Loader.setup
+
+require_relative "./lib/casting_config_helpers"
+require_relative "./config"
+
+require "rack/unreloader"
+
+Unreloader = Rack::Unreloader.new(
+  reload: Config.development?,
+  autoload: true,
```

Comment:
Does autoloading at production has any performance penalties?

Reply:
Probably a minor one in application startup I'd think. But this is how it's done with zeitwerk